projects/github-actions/repo-gardening/src/tasks/add-labels/index.js

```diff
@@ -218,6 +218,14 @@ async function getLabelsToAdd( octokit, owner, repo, number, isDraft, isRevert )
 			}
 		}
 
+		// External Media extension.
+		const externalMedia = file.match(
+			/^projects\/plugins\/jetpack\/extensions\/shared\/external-media\//
+		);
+		if ( externalMedia !== null ) {
```

Comment:
Not a blocker (and I see it's done like this elsewhere), but I prefer simplifying the construct: 
```suggestion
		if ( externalMedia ) {
```

Reply:
I'll see about updating them all at once in a future PR!